modules/ROOT/pages/_partials/sgw-svr-compatibility.adoc

```diff
@@ -52,18 +52,15 @@ endif::[]
 
 :fnref3-0: footnote:3-0[]
 
-
-include::partial$block-caveats.adoc[tags="cbs6.0ke-xattrs"]
-
 .Sync Gateway/Couchbase Server
 Compatibility Matrix
-[cols="^1,3,^1,^1,^1,^1,^1,^1,^1"]
+[cols="^1,3,^1,^1,^1,^1,^1,^1,^1,^1"]
```

Comment:
This might be out of scope for this PR, but I find table tricky to read. I'd be tempted to flip this so that the newest versions are first. This is especially useful for Sync Gateway versions, IMO.

Sync Gateway 4.0
Sync Gateway 3.3

Flipping the column names in reverse order also makes sense to me, but this should be the call of someone in docs or PM.

I found this confusing as a reader. We certainly do not care about Sync Gateway 1.5